apps/gitness/src/pages-v2/project/pull-request/pull-request-list.tsx

```diff
@@ -133,13 +144,15 @@ export default function PullRequestListPage() {
 
   return (
     <SandboxPullRequestListPage
+      isProjectLevel
```

Comment:
We should not require another prop to specify is the scope is `project` or `repo`.

Passing `spaceId` and not `repoId` props in `PullRequestPageProps` should mean Project scope itself.

This is the same logic to render `PR List` component at `Project` level.